src/api/journalEntries/journalEntry.ts

```diff
@@ -1,6 +1,6 @@
 import API from '../apolloClient';
 import { gql } from '@apollo/client';
-import { useQuery } from 'react-query';
+import { useQuery, useMutation, useQueryClient  } from 'react-query';
```

Comment:
The documentation for useMutation and useQueryClient were a bit confusing, but was able to figure it out through stackoverflow posts. Glad I was able to move these into the api section of the repo, creating a custom hook, so that they weren't being called in multiple components.